core/nodejsActionBase/app.js

```diff
@@ -18,22 +18,22 @@
 var config = {
         'port': 8080,
         'apiHost': process.env.__OW_API_HOST,
-        'allowConcurrent': process.env.__OW_ALLOW_CONCURRENT
+        'allowConcurrent': process.env.__OW_ALLOW_CONCURRENT,
+        'requestBodyLimit': "48mb"
```

Comment:
Should we support this configuration parameter as settable from an environment variable (with a default value)?